src/components/views/passwordReset/PasswordReset.spec.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '@Redux/store';
+import SubmitEmail, { SubmitEmailComponent } from './SubmitEmail';
+import NewPassword from './NewPassword';
```

Comment:
Update these paths

Reply:
Hello @mazma1  I don't understand what you mean by updating this path since the test file and the component are in the same folder.